### PR TITLE
Migration to create-react-app

### PR DESCRIPTION
**Pull request by antoniotrkdz**
Everything needs careful checking
relates to #47 sorry!

### CODE REVIEW

**Comment by des-des** [comment]
@antoniotrkdz You have commited your node modules

**Comment by antoniotrkdz** [comment]
Sorry, I realised after that I have node modules also in the src/server folder, which I forgot to add to .gitignore.
Can you still merge this? ignoring the modules?

**Comment by des-des** [comment]
I cannot review 600K lines of code

**Comment by des-des** [comment]
image

**Comment by antoniotrkdz** [comment]
Done removing the node modules

**Comment by des-des** [comment]
1. move server out of source
  2. merge the packages together
  3. test on heroku

File: react-ui/src/index.js [hunk 1]
```diff
@@ -1,3 +1,9 @@
+//import React from 'react';
```
Comment (des-des) [inline react-ui/src/index.js]: :fire:

**Pull request merged**